circleform.py

```diff
@@ -31,6 +32,10 @@ def isOver(self, x, y):
     def move(self, x, y):
         self._origin = (x, y)
 
+    def className(self):
```

Comment:
use property better idea

@property
def className(self):
   return ...

usage:
toto.className (without parenthesis). Try toto.className = "Forced name"
because you have a property with only a getter